<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="0ms"
		durationms="0"
		type="root"
		rid="6b157d84ac5b5a0"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="1"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		user="erdogb"
		rxversion="7.2.0.0"
		host="EC168"
		osversion="Windows 7 Service Pack 1 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="de-DE"
		dtnlanguage="de-CH"
		screenresolution="3840x1200"
		timestamp="20.04.2018 13:20:21"
		timestampiso="2018-04-20T11:20:21"
		endtime="20.04.2018 13:20:33"
		timeoutfactor="1"
		module="IOmoduleVerification">
		<detail>
			IOmoduleVerification
		</detail>
		<varbindings>
		</varbindings>
		<item
			timeRelativeToTestSuiteStartTime="00:11.220"
			timeRelativeToTestModuleStartTime="00:11.220"
			timeWallClock="13:20:33"
			time="00:11.220"
			level="Error"
			category="User"
			errimg=".\IOmodule_95fda495_6b0bf02c_trace.jpg"
			errthumb=".\IOmodule_95fda495_6b0bf02c_trace_thumb.jpg">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
